docs/configuration/vpn/wireguard.rst

```diff
@@ -71,26 +84,39 @@ you want to tunnel (allowed-ips) to configure a WireGuard tunnel. The
 public key below is always the public key from your peer, not your local
 one.
 
-**local side**
+**local side - commands**
 
 .. code-block:: none
 
-  set interfaces wireguard wg01 address '10.1.0.1/24'
+  set interfaces wireguard wg01 address '10.1.0.1/30'
   set interfaces wireguard wg01 description 'VPN-to-wg02'
-  set interfaces wireguard wg01 peer to-wg02 allowed-ips '10.2.0.0/24'
-  set interfaces wireguard wg01 peer to-wg02 address '192.168.0.142'
-  set interfaces wireguard wg01 peer to-wg02 port '12345'
+  set interfaces wireguard wg01 peer to-wg02 allowed-ips '192.168.2.0/24'
+  set interfaces wireguard wg01 peer to-wg02 address '2.2.2.2'
```

Comment:
please don't use public IP space in examples, when it is not necessary.
reference also [here](https://docs.vyos.io/en/latest/documentation.html#address-space)

Reply:
Sure, though I will make it clear that it is a public IP address. Something like <WAN Addr>
When I tried to configure this myself, it was unclear what IP address was the tunnel and what the WAN address was.